doc/modules/pipeline.rst

```diff
@@ -41,11 +41,11 @@ is an estimator object::
     >>> pipe = Pipeline(estimators)
     >>> pipe # doctest: +NORMALIZE_WHITESPACE
     Pipeline(steps=[('reduce_dim', PCA(copy=True, iterated_power='auto',
-    n_components=None, random_state=None, svd_solver='auto', tol=0.0,
-    whiten=False)), ('clf', SVC(C=1.0, cache_size=200, class_weight=None,
-    coef0=0.0, decision_function_shape='ovr', degree=3, gamma='auto',
-    kernel='rbf', max_iter=-1, probability=False, random_state=None,
-    shrinking=True, tol=0.001, verbose=False))])
+        n_components=None, random_state=None, svd_solver='auto', tol=0.0,
+        whiten=False)), ('clf', SVC(C=1.0, cache_size=200, class_weight=None,
+        coef0=0.0, decision_function_shape='ovr', degree=3, gamma='auto',
+        kernel='rbf', max_iter=-1, probability=False, random_state=None,
+        shrinking=True, tol=0.001, verbose=False))])
```

Comment:
that won't work because we would add a whitespace that is not present in the original string even with `+NORMALIZE_WHITESPACE`. @glemaitre's fix look good to me.